gnomad/resources/grch38/gnomad.py

```diff
@@ -503,43 +579,45 @@ def gnomad_gks(
             " please also specify 'by_ancestry_group' to stratify by."
         )
 
-    # Call and return add_gks_vrs and add_gks_va for chosen arguments.
-
     # Select relevant fields, checkpoint, and filter to interval before adding
     # annotations
-    ht = ht.annotate(
-        faf95=hl.rbind(
-            hl.sorted(
-                hl.array(
-                    [
-                        hl.struct(
-                            faf=ht.faf[ht.faf_index_dict[f"{pop}-adj"]].faf95,
-                            population=pop,
-                        )
-                        for pop in FAF_POPS
-                    ]
-                ),
-                key=lambda f: (-f.faf, f.population),
-            ),
-            lambda fafs: hl.if_else(
-                hl.len(fafs) > 0,
-                hl.struct(
-                    popmax=fafs[0].faf,
-                    popmax_population=hl.if_else(
-                        fafs[0].faf == 0, hl.missing(hl.tstr), fafs[0].population
-                    ),
-                ),
-                hl.struct(
-                    popmax=hl.missing(hl.tfloat), popmax_population=hl.missing(hl.tstr)
-                ),
-            ),
-        )
-    )
 
-    keep_fields = [ht.freq, ht.info.vrs, ht.faf95]
+    # Pull up LCR flag and make referrable in the same field
+    if high_level_version == "v3":  # v3
+        ht = ht.annotate(lcr=ht.region_flag.lcr)
+    else:  # v4
+        ht = ht.annotate(lcr=ht.region_flags.lcr)
+
+    # Pull up v3 / v4 allele balance histogram arrays
+    if high_level_version == "v3":  # v3
+        ht = ht.annotate(ab_hist_alt=ht.qual_hists.ab_hist_alt)
+    else:  # v4
+        ht = ht.annotate(ab_hist_alt=ht.histograms.qual_hists.ab_hist_alt)
+
+    if high_level_version == "v3":
+        ht = add_grpMaxFAF95_v3(ht)
+    else:
+        ht = add_grpMaxFAF95_v4(ht)
```

Comment:
```suggestion
    ht = ht.annotate(ab_hist_alt=ht.histograms.qual_hists.ab_hist_alt)

    ht = add_grpMaxFAF95_v4(ht)
```

Reply:
appropriate change if fully not supporting v3 at all - may need beautified with some more comments or documentation around here, or running black on it again